Flaky test in pex

```
[2018-04-22T04:42:37Z] ERROR [pex]: corrupt or empty file, rewriting file path="/tmp/peers.txt420390835"
[2018-04-22T04:42:38Z] ERROR [pex]: Invalid address in peers JSON file 112.32.32.14:1: Port must be >= 1024
[2018-04-22T04:42:38Z] ERROR [pex]: Invalid address 112.32.32.14:1: Port must be >= 1024
[2018-04-22T04:42:38Z] WARN [pex]: Add peers failed, peer list is full
[2018-04-22T04:42:38Z] INFO [pex]: Add peers sees an invalid address 112.32.32.14:1: Port must be >= 1024
[2018-04-22T04:42:38Z] INFO [pex]: Reset all peer's retry times
--- FAIL: TestPexIncreaseRetryTimes (0.00s)
    --- FAIL: TestPexIncreaseRetryTimes/ok (0.00s)
	Error Trace:	pex_test.go:1024
	Error:      	Not equal: 
	            	expected: pex.Peer{Addr:"112.32.32.14:7200", LastSeen:1524372158, Private:false, Trusted:false, HasIncomingPort:false, RetryTimes:1}
	            	actual: pex.Peer{Addr:"112.32.32.14:7200", LastSeen:1524372159, Private:false, Trusted:false, HasIncomingPort:false, RetryTimes:1}
	            	
	            	Diff:
[2018-04-22T04:42:39Z] ERROR [pex]: Remote peers list has invalid address #comment: Invalid address
[2018-04-22T04:42:39Z] ERROR [pex]: Remote peers list has invalid address 127.0.0.1:8080: Localhost address is not allowed
[2018-04-22T04:42:39Z] ERROR [pex]: Remote peers list has invalid address 11.33.11.33: Invalid address
[2018-04-22T04:42:39Z] ERROR [pex]: Remote peers list has invalid address 22.44.22.44:99: Port must be >= 1024
FAIL
FAIL	github.com/skycoin/skycoin/src/daemon/pex	2.021s
```

--- a/src/daemon/pex/pex_test.go
+++ b/src/daemon/pex/pex_test.go
@@ -1022,7 +1022,7 @@ func TestPexIncreaseRetryTimes(t *testing.T) {
 				p, ok := pex.peerlist.peers[k]
 				require.True(t, ok)
 				if p.LastSeen != 0 {
-					require.True(t, p.LastSeen-v.LastSeen <= 2)
+					require.InDelta(t, v.LastSeen, p.LastSeen, 2)
 					p.LastSeen = 0
 					v.LastSeen = 0
 				}

--- a/src/daemon/pex/pex_test.go
+++ b/src/daemon/pex/pex_test.go
@@ -1021,6 +1021,11 @@ func TestPexIncreaseRetryTimes(t *testing.T) {
 			for k, v := range tc.expect {
 				p, ok := pex.peerlist.peers[k]
 				require.True(t, ok)
+				if p.LastSeen != 0 {
+					require.True(t, p.LastSeen-v.LastSeen <= 2)
+					p.LastSeen = 0
+					v.LastSeen = 0
+				}
 				require.Equal(t, v, *p)
 			}
 		})

